scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -60,25 +65,33 @@
 
 # standard regular axes
 axis_eta = hist.axis.Regular(template_neta, template_mineta, template_maxeta, name = "eta", overflow=not args.excludeFlow, underflow=not args.excludeFlow)
-if not args.vqt3dsmoothing:
-    axis_pt = hist.axis.Regular(template_npt, template_minpt, template_maxpt, name = "pt", overflow=not args.excludeFlow, underflow=not args.excludeFlow)
-else :
-    axis_pt_list = [24.,26.,28.,30.,32.,34.,36.,38.,40., 42., 44., 47., 50., 55., 60., 65.]
-    axis_pt = hist.axis.Variable(axis_pt_list, name = "pt")
+## MARCO temporarily commented out code
```

Comment:
Do you really want to PR all the commented out lines? Can't it just be removed, or refactored in a nicer way?

Reply:
So, PR #257 is actually fixing this, I can backport to this PR too now that it is more clear what we need to keep and what not

Reply:
Won't there be a conflict between the two if you comment it out in one and delete it in the other? I would just delete it here then fix it there. I imagine this PR should be merged first

Reply:
Done